apis/python/src/tiledbsoma/experiment_query/anndata.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict, Tuple
+
+import anndata
+import pyarrow as pa
+import scipy.sparse as sparse
+
+from .types import ExperimentQueryReadArrowResult
+
+
+def arrow_to_scipy_csr(X: pa.Table, shape: Tuple[int, int]) -> sparse.csr_matrix:
+    return sparse.csr_matrix(
+        (X["soma_data"].to_numpy(), (X["_dim_0"].to_numpy(), X["_dim_1"].to_numpy())),
```

Comment:
```
        (X["soma_data"].to_numpy(), (X["soma_dim_0"].to_numpy(), X["soma_dim_1"].to_numpy())),
```
?

Reply:
Can you expand on `?`?  Happy to discuss, but not sure what the question is :-)

Reply:
@bkmartinjr sorry I was recommending you replace

```
        (X["soma_data"].to_numpy(), (X["_dim_0"].to_numpy(), X["_dim_1"].to_numpy())),
```

with

```
        (X["soma_data"].to_numpy(), (X["soma_dim_0"].to_numpy(), X["soma_dim_1"].to_numpy())),
```

Reply:
Actually, this code is correct as written.  See query.py::_rewrite_X_for_positional_indexing().

the idea is that AnnData requires positional indexing, not joinids.